### DISCUSSION TITLE
Spotlight import support

### DISCUSSION THREAD

**Discussion by sand0u** [mitsuba-renderer/mitsuba-blender#68] [Q&A]
I noticed that there is currently no import function that supports spotLight, is it possible to support it, or can I get this function by modifying the source code myself? Thanks

**Comment by bathal1** [comment]
Hi,
we already support *export* [here](https://github.com/mitsuba-renderer/mitsuba-blender/blob/3336eeca316fe617689bd1020c53a442acd5221c/mitsuba-blender/io/exporter/lights.py#L84-L101), so importing should in theory be feasible. PRs are welcome !
